bazel/v8.patch

```diff
@@ -7,13 +7,15 @@
 # 7. Fix build errors in SIMD IndexOf/includes (https://crrev.com/c/3749192).
 # 8. Fix build on arm64.
 # 9. Fix build on older versions of Linux.
+# 10. Fix MemoryAllocator::PartialFreeMemory() which shouldn't try to change permissions of RWX pages,
+#     mainly affecting macOS on Apple silicon (https://chromium-review.googlesource.com/c/v8/v8/+/3700352).
```

Comment:
Can you file an issue to track when this is released?

Reply:
Tracked here: https://github.com/envoyproxy/envoy/issues/23258.